p2p/src/interface/types.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2023 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO: Implement From PeerContext for ConnectedPeer
```

Comment:
I think it's PeerInfo we're interested in, not PeerContext

Reply:
`p2p::peer_manager::peerdb::PeerContext` contains some interesting stuff (address and ban score for example). We probably can move some of it to `PeerInfo`.